### PR TITLE
add serializable for VersionInfo, add VersionName to manifest

### PR DESCRIPTION
**Pull request by superyyrrzz**

### CODE REVIEW

File: src/Microsoft.DocAsCode.Plugins/OutputFileInfo.cs [hunk 1]
```diff
@@ -26,6 +26,9 @@ public string RelativePath
             }
         }
 
+        [JsonProperty("version_name")]
+        public string VersionName { get; set; }
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Plugins/OutputFileInfo.cs]: >VersionName [](start = 22, length = 11)

is it possible to add a mapping in root manifest?

File: src/docfx/Models/VersionConfig.cs [hunk 2]
```diff
@@ -3,9 +3,12 @@
 
 namespace Microsoft.DocAsCode
```
Comment (vwxyzh) [inline src/docfx/Models/VersionConfig.cs]: namespace

File: src/Microsoft.DocAsCode.Plugins/Manifest.cs [hunk 3]
```diff
@@ -45,6 +45,9 @@ public Manifest(IEnumerable<ManifestItem> files)
         [JsonProperty("incremental_info")]
         public List<IncrementalInfo> IncrementalInfo { get; set; }
 
+        [JsonProperty("version_info")]
+        public Dictionary<string, VersionInfo> VersionInfo { get; set; }
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Plugins/Manifest.cs]: >VersionInfo [](start = 47, length = 11)

VersionInfos?

Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Plugins/Manifest.cs]: want to align with `IncrementalInfo` above

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**